src/main/java/de/cas/mse/exercise/SumDigPower.java

```diff
@@ -5,37 +5,47 @@
 
 public class SumDigPower {
 
-	public List<Long> sumDigPow(long a, long b) {
+    List<Long> eurekaList = new ArrayList<Long>();
```

Comment:
I dont think that should be a global variable, when you would call the function twice you would get invalid results